kernel/packages/decentraland-ecs/src/decentraland/math/Quaternion.ts

```diff
@@ -483,33 +484,24 @@ export class Quaternion {
 
   /**
    * Creates a rotation which rotates from fromDirection to toDirection.
-   * @param from - defines the first Vector
-   * @param to - defines the second Vector
+   * @param from - defines the first direction Vector
+   * @param to - defines the target direction Vector
+   * Unity-based calculations implemented from https://forum.unity.com/threads/quaternion-lookrotation-around-an-axis.608470/#post-4069888
```

Comment:
Can we leave unity related comment outside method documentation?

Reply:
Done